lib/posts.js

```diff
@@ -220,4 +221,48 @@ PostsRequest.prototype.statuses = function() {
 	return this;
 };
 
+/**
+ * @method year
+ * @chainable
+ * @param year - Integer for year
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.year = function ( year ) {
+  return this.filter( 'year', year );
+};
+
+/**
+ * @method month
+ * @chainable
+ * @param month - Integer for month or month string ("january")
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.month = function ( month ) {
+  if ( _.isString( month ) ) {
+    month = new Date( Date.parse(month +" 1, 2012") );
+
+    if ( isNaN(month) ) {
+      return this;
+    }
+
+    month = month.getMonth() + 1;
```

Comment:
This also needs a comment to clarify the different indexing (0-based vs 1-based) between JS dates and WP_Query.